templates/viewer.html

```diff
@@ -1,18 +1,10 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-	<meta charset="utf-8">
-	<title>Napari Monitor</title>
-</head>
-
-<body style="margin:0; padding:0; overflow:hidden">
-
-
-	<!-- This will contain the canvas -->
-	<div id="WebGLContainer"></div>
-
-	<script src="/static/bundle.js"></script>
-</body>
-
-</html>
+{% extends "base.html" %}
+{% block content %}
+<div id="WebGLContainer"></div>
+
+<script type="module">
+	import { startViewer, connectSocketInput } from '/static/viewer.js';
```

Comment:
With in a script module tag, you can import any module exports from your bundle(s).

Reply:
Cool.